src/test/java/teammates/test/cases/browsertests/AdminHomePageUiTest.java

```diff
@@ -86,12 +86,12 @@ private void testCreateInstructorAction() throws Exception {
         BackDoor.deleteCourse(demoCourseId);
         BackDoor.deleteInstructor(demoCourseId, instructor.email);
         homePage.createInstructorByInstructorDetailsSingleLineForm(instructorDetails);
```

Comment:
You don't mind to refresh again right? Just kidding. That the point I want to find out. Is it possible to do this? Also, if we successfully break down the test cases, we could write our own `TestRunner` to retry the specific test cases. (similar to what the gradle script does currently)